glitter/block_admin.py

```diff
@@ -229,6 +231,107 @@ def response_change(self, request, obj):
         # Update column and close popup - don't bother with a message as they won't see it
         return self.response_rerender(request, obj, 'admin/glitter/update_column.html')
 
+    def formfield_for_dbfield(self, db_field, **kwargs):
```

Comment:
Why is this method required? It's exact the same functionality as in `contrib/admin/options.py` can't see any difference between them.